packages/cli/lib/commands/create.js

```diff
@@ -59,7 +59,10 @@ module.exports = async (name, starter = 'default') => {
     {
       title: `Install dependencies`,
       task: (_, task) => {
-        const command = useYarn ? 'yarn' : 'npm'
+        let command = 'npm'
+        if (!fs.existsSync(`${dir}/package-lock.json`)) {
```

Comment:
```suggestion
        if (!fs.existsSync(path.join(dir, 'package-lock.json'))) {
```
Using template literals to merge paths caused some issues on Windows for me a while ago. So I think we should use the `path` module to get the correct path. But I don't have Windows available atm, so I can't test it...